src/SDKs/DataFactory/DataFactory.Tests/ScenarioTests/DataFactoryScenarioTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using DataFactory.Tests.Utils;
+using Microsoft.Azure.Management.DataFactory;
+using Microsoft.Azure.Management.DataFactory.Models;
+using Microsoft.Rest.Azure;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace DataFactory.Tests.ScenarioTests
+{
+    public class DataFactoryScenarioTests : ScenarioTestBase<DataFactoryScenarioTests>
+    {
+        [Fact]
+        [Trait(TraitName.TestType, TestType.Scenario)]
```

Comment:
@bgold09 just FYI, we have not started using traits in SDK tests yet. All tests are executed as of now. But if you would like to categorize tests, go ahead do it. But please be aware the categories can be different when we introduce categories for the entire repo.

Reply:
sounds good, we were using these only for organizational purposes within our own test project